### PR TITLE
SDK Version Header

### LINKED ISSUE #14: Add Adzerk SDK version header
## Background
The four new Decision SDKs (Java, JavaScript, Ruby, & Python) transmit an `X-Adzerk-Sdk-Version` header to the ad server & UserDB server to let us know the SDK name and version.  We need this telemetry from the iOS and Android SDKs too.  This improves Adzerk's ability to debug customers' API use as well as plan improvements to the SDKs.

## Acceptance Criteria
- The SDK should send an SDK header on all ad server and UserDB requests showing it's the iOS SDK and its package version.
- This header will use the `X-Adzerk-Sdk-Version` header.
- This header will have a payload of: `[github-repo-name]:[package-version]`
  - example: `adzerk-ios-sdk:1.1`

### PR DESCRIPTION
**Pull request by subdigital**
This adds `X-Adzerk-Sdk-Version` header to all API requests.

Fixed #14

### CODE REVIEW

File: AdzerkSDK/AdzerkSDK/AdzerkSDK.swift [hunk 1]
```diff
@@ -499,6 +499,7 @@ public typealias ADZUserDBUserResponseCallback = (ADZUser?, Error?) -> ()
         let config = URLSessionConfiguration.ephemeral
         config.httpAdditionalHeaders = [
             "User-Agent" : UserAgentProvider.instance.userAgent,
+            "X-Adzerk-Sdk-Version": "adzerk-ios-sdk:\(AdzerkSDKVersion)",
```
Comment (CrshOverride) [inline AdzerkSDK/AdzerkSDK/AdzerkSDK.swift:502]: How does the SDK Version variable get set here?

Comment (subdigital) [reply AdzerkSDK/AdzerkSDK/AdzerkSDK.swift:502]: That is a static variable defined in the same file. We don't have any automation around this at the moment, it must be manually set.

Comment (CrshOverride) [reply AdzerkSDK/AdzerkSDK/AdzerkSDK.swift:502]: @subdigital Is there an "easy" way to automate it at build-time or anything? Also, I just realized that the key should be `adzerk-decision-sdk-ios` to match the pattern of the other SDKs (we're releasing both decision and management in the future so knowing which in the string is valuable).

Comment (subdigital) [reply AdzerkSDK/AdzerkSDK/AdzerkSDK.swift:502]: I can add a script to modify this file w/ the bundle version at build time. I'll also fix that key.

**Pull request merged**